src/modal/alert-modal.component.ts

```diff
@@ -124,4 +126,11 @@ export class AlertModal extends BaseModal implements AfterViewInit {
 		}
 		this.closeModal();
 	}
+
+	handleClickOverlay() {
+		if (this.onOverlayClick === "override") {
```

Comment:
This gets the job done, but I'm concerned that we now have two different ways to handle controlling how a modal closes that do very different things and could lead to unexpected bugs on the consumer side. We already have a `close` function that controls how/when the modal closes, so it may be clearer to take advantage of that over introducing new properties and methods

Reply:
Utilizing existing functions is definitely better than introducing new methods, I should have considered this more carefully :p 
To clarify, the trigger parameter in the close signature suggested would be part of user input?

Reply:
In the case of `alert-modal` the overlay click and the close button click are handled the same way, so we need a way for users to differentiate 👍